core/server/data/schema/schema.js

```diff
@@ -390,6 +390,15 @@ module.exports = {
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
     },
+    members_stripe_info: {
```

Comment:
Soz, have a lot of whys in this PR :grimacing: This is a first "3-part" table name in Ghost and wondering what's the reason behind adding the `_info` postfix. Do you envision `memebers_stripe` being a thing in a very near future?

Reply:
No probs! This is trying to be specific about what the table stores e.g. "stripe info" for "members". I don't think we'll have a `members_stripe` in the very near future - but it's possible we would :man_shrugging: